### ISSUE TITLE
Longer Context for OpenAI and Superbooga Extensions

### ISSUE THREAD

**Issue by chigkim** [oobabooga/text-generation-webui#2911]
Since we can now have up to 8192 context length with SuperHot models and Exllama, can we have OpenAI and Superbooga extensions to support                                                                                                                    long context as well?

**Comment by gptzerozero** [comment]
Do you require Exllama if you want to extend the context length to 8192? Or is it sufficient to load a superhot-8k model?

**Comment by chigkim** [comment]
I just use the default config.yaml that comes with oobabooga, an it does have that lines.

```
.*superhot-8k:
  truncation_length: 8192
```

Context length works fine in the web gui, it just doesn't work in OpenAI extension.

`Warning: too many messages for context size, dropping 1 oldest message(s).`

**Comment by matatonic** [comment]
Yeah, you are correct, the setting is making it into the GUI but the shared.settings structure isn't being updated with the new length. Looking into it...

**Comment by matatonic** [comment]
> Do you require Exllama if you want to extend the context length to 8192? Or is it sufficient to load a superhot-8k model?

The superhot-8k models are trained to generate more, but it does require the compressed embeddings in exllama to make use of an 8k context as afar as I am aware.
